patches/node/fix_-wnonnull_warning.patch

```diff
@@ -0,0 +1,41 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Charles Kerr <charles@charleskerr.com>
+Date: Thu, 6 Mar 2025 19:31:29 -0600
+Subject: fix: -Wnonnull warning
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Fixes this warning:
+
+> 2025-03-07T01:05:01.8637705Z ../../third_party/electron_node/src/debug_utils.cc(257,12): error: null passed to a callee that requires a non-null argument [-Werror,-Wnonnull]
+> 2025-03-07T01:05:01.8638267Z   257 |     return nullptr;
+> 2025-03-07T01:05:01.8638481Z       |            ^~~~~~~
+> 2025-03-07T01:05:01.8638700Z 1 error generated.
+
+Not sure why this warning was never triggered before; `git blame`
+indicates this code hasn't changed in ages:
```

Comment:
Looks like `std::string(nullptr)` was undefined behavior prior to C++23 which closed the gap. The Chromium folks [were discussing that particular change](https://groups.google.com/a/chromium.org/g/cxx/c/m6g325zAApc) and after they discussed with libc++ folks on Discord it was updated to start warning about it.